examples/chart/teleport-cluster/README.md

```diff
@@ -4,6 +4,9 @@ This chart sets up a single node Teleport cluster.
 It uses a persistent volume claim for storage.
 Great for getting started with Teleport.
 
+It can also deploy HA Teleport clusters on EKS, GKE and other Kubernetes clusters.
+See our [guides](https://goteleport.com/docs/kubernetes-access/helm/guides) for details.
```

Comment:
Please make this a relative link (esp. when porting/backporting) otherwise it'll point to the current version. 

Also, I'd prefer this to link to the local documentation rather than the hosted docs.

Reply:
This isn't in the docs, it's in the README file (which is under `examples`)

Reply:
If you prefer it to go to the current hosted version that's fine (since that's defaulted to at that URL as you know). 

I think every README is versioned too? So, if someone were reading this locally, clicking on the relative link would open the local, versioned, document?

Reply:
Clicking a relative link would indeed open the docs MDX file for the same branch, but there'd be no formatting, images or anything else which would basically render the guides unusable. I think it's smart here just to point to the hosted version at all times.